doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2613,14 +2613,14 @@ NCSDK-15512: Modem traces retrieval incompatible with TF-M
 Libraries for networking
```

Comment:
Why are all other sections called
```
Modem libraries
Bluetooth libraries
Binary libraries
...
```
and this is one called
`Libraries for networking` (instead of `Networking libraries`) ?

Reply:
"Libraries for networking" is mentioned because same term is used in the docs - https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/networking/index.html

Reply:
ok, thanks